tutorials/introductory/images.py

```diff
@@ -205,20 +199,19 @@
 #
 # You can specify the clim in the call to ``plot``.
```

Comment:
And it`s `imshow` not `plot`, we are talking about.

Then maybe:

```suggestion
# You can specify the colormap limits using ``imshow(..., clim=...)``.
```

Sorry, to comment on code you're not responsible for, but this is too glaringly wrong.